prototype_source/pt2e_quant_ptq_x86_inductor.rst

```diff
@@ -161,7 +162,15 @@ After these steps, we finished running the quantization flow and we will get the
 3. Lower into Inductor
 ------------------------
 
-After we get the quantized model, we will further lower it to the inductor backend.
+After we get the quantized model, we will further lower it to the inductor backend. The default Inductor wrapper
+generates Python code to invoke both generated kernels and external kernels. Additionally, Inductor supports a C++ wrapper
```

Comment:
can you also talk about what this enables people to do? I assume this allows people to deploy the model to C++ environment now? and is there any additional code examples for that?

Reply:
I believe the C++ Wrapper can help reduce Python overhead, but currently, it alone cannot achieve pure C++ deployment. The plan to implement pure C++ deployment should involve AOTInductor, which is based on C++ Wrapper. cc @chunyuan-w.

Reply:
Modify it by:

- Add the benefits of C++ wrapper: for now it can reduce python overhead; in the further, based on it, we can implement the Pure C++ deployment.
- Add the link of dedicated C++ wrapper tutorial for further study.

@jerryzh168 please kindly help to take a look again.

Reply:
thanks, is there a timeline to support AOTInductor?

Reply:
I think the current blocking issue is related to the serialization of oneDNN tensor. cc @chunyuan-w @jgong5 for more comments of the detail timeline.

Reply:
Hi just curious for this to see if there is any update.  Can we do with AOTInductor now?

cc @jerryzh168 @muchulee8

Reply:
Hi @hl475, I am afraid `AOTInductor` still can't gives good the performance in this case for now.

Reply:
Thanks @leslie-fang-intel , so you mean X86InductorQuantizer works with AOTInductor now, but the perf is not optimized, right?

If so, wondering if you have any benchmarks could share or understand what exactly do you mean "can't give good the performance"?

Reply:
Hi @hl475, I think they can work together. But you will run with fp32 conv/linear actually. To explain more, because of this issue: https://github.com/pytorch/pytorch/issues/114450. You can't turn on freeze when using `AOTInductor`. So, we are not able to lower the `q->conv/linear->dq` into `qconv/qlinear`.